wger/exercises/api/views.py

```diff
@@ -130,6 +132,45 @@ def search(request):
     return Response(json_response)
 
 
+class ExerciseInfo(APIView):
+
+    def get(self, request):
+        """Returns all the information about an exercise."""
+        """
+        It uses the same functionality as search.
+        It takes the exercise name as the parameter
+        and returns all the information about that exercise
+        """
+        exercise_name = request.GET.get('term', None)
+        results = []
+        json_response = {}
+
+        if exercise_name:
+            exercises = (
+                Exercise.objects.filter(name__icontains=exercise_name)
```

Comment:
I think you could create a helper file to place this long filter query and just import, store it in a variable and just use it.